src/app/services/BD/type/Floors.ts

```diff
@@ -0,0 +1,16 @@
+export type Floors = {
+  [key: string]: Array<{
+    id: string;
+    number: number;
+    places: Array<{
+      label: string;
+      type: string;
+      placeStatus: {
+        blocked: boolean;
+        occupants: string;
+        start: number;
+        end: number;
```

Comment:
+ coordinates of the place on plan

Reply:
Add new key [coordinates: string] in object placeStatus ?

Reply:
yes. check SVG `<rect>`'s attributes - they need to be stored to properly highlight working place on the image.